src/app/tests/suites/TestGroupsCluster.yaml

```
@@ -0,0 +1,358 @@
+          values:
+              - name: "groupId"
+                value: 0x1111
+      response:
+          values:
+              - name: "status"
+                value: 0
+              - name: "groupId"
+                value: 0x1111
+
+    - label: "View Group 1 (not removed)"
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: "groupId"
+                value: 0x0001
+      response:
+          values:
+              - name: "status"
+                value: 0
+              - name: "groupId"
+                value: 0x0001
+              - name: "groupName"
+                value: "Group #1"
+
+    - label: "View Group 2 (removed)"
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: "groupId"
+                value: 0x1111
+      response:
+          values:
+              - name: "status"
+                value: 0x8B
+              - name: "groupId"
+                value: 0x1111
+
+    - label: "View Group 3 (not removed)"
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: "groupId"
+                value: 0x7fff
+      response:
+          values:
+              - name: "status"
+                value: 0
+              - name: "groupId"
+                value: 0x7fff
+              - name: "groupName"
+                value: "Group #3"
+
+    - label: "Get Group Membership 2"
+      command: "GetGroupMembership"
+      arguments:
+          values:
+              - name: "groupList"
+                value: [0x01, 0x02, 0x1111, 0x03, 0x7fff]
+      response:
+          values:
+              - name: "capacity"
+                value: 0xff
+              - name: "groupList"
+                value: [0x01, 0x7fff]
+
+    - label: "Remove All"
+      command: "RemoveAllGroups"
+      response:
+          error: 0x00
+
+    - label: "View Group 1 (removed)"
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: "groupId"
+                value: 0x0001
+      response:
+          values:
+              - name: "status"
+                value: 0x8B
+              - name: "groupId"
+                value: 0x0001
+
+    - label: "View Group 2 (still removed)"
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: "groupId"
+                value: 0x1111
+      response:
+          values:
+              - name: "status"
+                value: 0x8B
+              - name: "groupId"
+                value: 0x1111
+
+    - label: "View Group 3 (removed)"
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: "groupId"
+                value: 0x7fff
+      response:
+          values:
+              - name: "status"
+                value: 0x8B
+              - name: "groupId"
+                value: 0x7fff
+    # TODO: Code generation fails when the list is empty
+    # - label: "Get Group Membership 3"
+    #   command: "GetGroupMembership"
```

You don't have to comment the test out.  Just do:
```suggestion
    - label: "Get Group Membership 3"
      disabled: true
      command: "GetGroupMembership"
```

---

I have uncommented the empty list tests, and disable them instead.